package.json

```diff
@@ -22,7 +22,8 @@
     "serve-favicon": "~2.4.5",
     "socket.io": "^2.0.4",
     "solc": "^0.4.19",
-    "web3": "~0.19.0"
+    "web3": "~0.19.0",
+    "zsl-web3.js": "^0.21.14"
```

Comment:
commit comment is not correctly described it's changes

config.json fix and `zsl-web3.js` changes are mixed should be separated.